1.9/overview/architecture/index.md

```diff
@@ -0,0 +1,29 @@
+---
+post_title: Architecture
+nav_title: Architecture
+menu_order: 2
+---
+
+DC/OS is a software platform on which user software runs. This platform layer runs on top of the infrastructure layer. The infrastructure layer is composed of virtual or physical hardware including compute, storage, and networking.
```

Comment:
the abstraction here is pretty poorly defined... The concepts page talks about Jobs & Services as the high level primitives DC/OS supports. But infinite arbitrary custom task types that can be added using Custom Service Schedulers. So it's inaccurate to label that layer as just microservices or just services... I suppose i could call it the "Service & Job Layer", but it's really like any arbitrary user software that runs as tasks.